high_card.rb

```diff
@@ -1,5 +1,117 @@
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
-players = []
+$winning_card = nil
```

Comment:
I understand its not a problem with this assignment, but I would caution against using global variables moving forward.